### ISSUE TITLE
The `add-path` command is deprecated and will be disabled soon

### ISSUE THREAD

**Issue by CallumHibbert** [autero1/action-terragrunt#5]
Warning from the Action:

```
[INFO] TerragruntVersion: v0.23.14
[INFO] Setting up Terragrunt version: 'v0.23.14'
[INFO] Downloading from: 'https://github.com/gruntwork-io/terragrunt/releases/download/v0.23.14/terragrunt_linux_amd64'
Warning: The `add-path` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
[INFO] Terragrunt version: 'v0.23.14' has been cached at /opt/hostedtoolcache/terragrunt/0.23.14/x64/terragrunt
```

Please see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/

Thanks.

**Comment by peter-murray** [comment]
Due to the urgent nature of resolving this if you are using this GitHub Action in your workflows and the pending 16th November breaking change on the CVE I have had to hard fork this project and publish the action to the marketplace from here; https://github.com/peter-murray/terragrunt-github-action

**Comment by CallumHibbert** [comment]
@peter-murray thanks very much for doing that.

**Comment by autero1** [comment]
Sorry for the inconvenience caused by delayed fix. This issue has been fixed in #7 and the latest release is now at `v1.0.0`.
